src/scripts/main.js

```diff
@@ -2,7 +2,17 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('tbody');
```

Comment:
The table with class 'dashboard' should be selected instead of 'tbody'. Use 'document.querySelector('.dashboard')' to select the correct element.